thunder/distributed/transforms/fsdp_v2.py

```diff
@@ -124,6 +124,7 @@ def transform_module(
         cd = get_compile_data(thunder_model)
         # TODO: promote use_fsdp and use_ddp to public members of CompileData
         cd.use_fsdp = True
+        cd.process_group_for_ddp = self.process_group
```

Comment:
ddp_v2 also has it - https://github.com/Lightning-AI/lightning-thunder/blob/52ee541073ddf6dce78b77ddb272621f712e3c6a/thunder/distributed/transforms/ddp_v2.py#L42

And TE executor needs it
https://github.com/Lightning-AI/lightning-thunder/blob/52ee541073ddf6dce78b77ddb272621f712e3c6a/thunder/executors/transformer_engineex.py#L180

Reply:
Hmhm. Not sure if cd is a good place for it, but let's do it for now.